js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -10,6 +10,11 @@ console.clear();
 */
 
 // --v-- write your code here --v--
+const title = "SocialMediaPost";
+const text = "hallo my name is Moritz";
```

Comment:
"Hallo" is a German word. Also, when you start a new sentence words start with capital letters.

Reply:
thanks